### PR TITLE
Allow init to respect the state directory and kubeconfig paths

### PR DESCRIPTION
**Pull request by thefirstofthe300**
I'm working on automating the install of AWS IAM authenticator as much as possible. It would be nice if init would just place the certificates in the correct place on boot.

### CODE REVIEW

**Comment by nckturner** [comment]
This makes a ton of sense to me.  Going back all the way to: https://github.com/kubernetes-sigs/aws-iam-authenticator/pull/10 it seems the idea is that there is some external process which needs to move stuff around.  But if we allow init to place stuff in the correct spot, then we can do init from an initContainer or something.  Can you please sign the CLA?

**Comment by richzli** [comment]
I ran into issues with this while trying to manually set up a cluster, as well.

From the documentation in the repository's README, it does seem like that `init` should be putting the generated files into their respective locations automatically. It'd simplify the process a lot if we didn't have to move the files ourselves afterwards.

**Comment by thefirstofthe300** [comment]
I need to get approval from my company on this. I don't expect any issues, but it may take a couple days.

**Comment by nckturner** [comment]
@thefirstofthe300 I'd like to merge this, can you get the CLA signed today?

File: cmd/aws-iam-authenticator/init.go [hunk 1]
```diff
@@ -36,19 +36,15 @@ var initCmd = &cobra.Command{
...
+		err = cfg.GenerateFiles()
 		if err != nil {
 			fmt.Fprintf(os.Stderr, "could not initialize: %v\n", err)
 			os.Exit(1)
 		}
 
-		logrus.Infof("copy %s to %s on kubernetes master node(s)", localCfg.CertPath(), cfg.CertPath())
-		logrus.Infof("copy %s to %s on kubernetes master node(s)", localCfg.KeyPath(), cfg.KeyPath())
-		logrus.Infof("copy %s to %s on kubernetes master node(s)", localCfg.GenerateKubeconfigPath, cfg.GenerateKubeconfigPath)
+		logrus.Infof("copy %s to %s on kubernetes master node(s)", cfg.CertPath(), cfg.CertPath())
```
Comment (nckturner) [inline cmd/aws-iam-authenticator/init.go:45]: This log line doesn't make sense anymore.

**Comment by nckturner** [comment]
So can you address how this change would affect a user who is relying on the current behavior of generating certificates/kubeconfig in the current directory?

**Comment by nckturner** [comment]
Per @wongma7's suggestion, in order to maintain backwards compatibility with the current behavior of the init command, we could generate them in the current directory *and* copy them to the desired location.  Or another option would be to add a feature flag that is default disabled (and warns of deprecated behavior), but when the flag is enabled then we get the desired behavior above.

**Pull request closed without merging**